plugins/generator-1.19.4/forge-1.19.4/templates/projectile/projectile.java.ftl

```diff
@@ -141,22 +135,24 @@ public class ${name}Entity extends AbstractArrow implements ItemSupplier {
 			this.discard();
 	}
 
+	public static ${name}Entity shoot(Level world, LivingEntity entity, RandomSource source) {
+		return shoot(world, entity, source, ${data.power}f, ${data.damage}, ${data.knockback});
+	}
+
 	public static ${name}Entity shoot(Level world, LivingEntity entity, RandomSource random, float power, double damage, int knockback) {
 		${name}Entity entityarrow = new ${name}Entity(${JavaModName}Entities.${data.getModElement().getRegistryNameUpper()}.get(), entity, world);
 		entityarrow.shoot(entity.getViewVector(1).x, entity.getViewVector(1).y, entity.getViewVector(1).z, power * 2, 0);
 		entityarrow.setSilent(true);
-		entityarrow.setCritArrow(${data.bulletParticles});
+		entityarrow.setCritArrow(${data.showParticles});
 		entityarrow.setBaseDamage(damage);
 		entityarrow.setKnockback(knockback);
-		<#if data.bulletIgnitesFire>
+		<#if data.igniteFire>
 			entityarrow.setSecondsOnFire(100);
 		</#if>
 		world.addFreshEntity(entityarrow);
 
-		<#if data.actionSound.toString()?has_content>

```

Comment:
Before, action sound was optional, and this should stay so (no call to this if not needed)